### ISSUE TITLE
Suggestion: adding a separate autoAF position

### ISSUE THREAD

**Issue by hamid13r** [eisfabian/PACEtomo#33]
Dear Fabian,

I am sorry if this suggestion is repeated or you have already thought about it. For our Krios, the only way of getting a consistent defocus at higher tilts is setting `addAF = True` and putting extra dose on the tracking position. 

Is there a way to define an autofocus position off the first target so we don't have to sacrifice that one target (per lamella)? This could improve our throughput even further!

Thanks in advance. 

Hamid

**Comment by eisfabian** [comment]
Dear Hamid,

You can use SerialEM's Low Dose Focus position offset to offset the focus position along the tilt axis. This should have no effect on PACEtomo as long as there are no targets in that area.

What are your defocus requirements and what kind of defocus spread can you achieve without `addAF`? If you have the defocus estimations of some of your tilt series collected without `addAF`, I would love to have a look to see if I can see any systematic issues.

Would it be interesting to have the option to turn `addAF` on only for higher tilt angles? I might consider this for a future version then.

Thanks again and best wishes,
Fabian

**Comment by hamid13r** [comment]
Thank you, I think the focus position in SerialEM would work great. Do i need to set it when I am on the item with "A" (acquire) on it?

Without `addAF` the defocus changes around 2-3 microns at high tilt I believe. I might be able to find our older data with defocus values.

**Comment by eisfabian** [comment]
Yes, you should be able to set it for each PACEtomo acquisition on the tracking target ("A") or use the general setting for all PACEtomo areas.
